kong/plugins/aws-lambda/CHANGELOG.md

```diff
@@ -15,6 +15,8 @@
 ## unreleased
 
 - fix: handle multivalueheaders [#59](https://github.com/Kong/kong-plugin-aws-lambda/pull/59)
+- feat: add support for detecting region from AWS_REGION or AWS_DEFAULT_REGION environment
```

Comment:
nit: we aren't maintaining the plugins changelogs -- they are there for reference. In fact, I think we should remove them entirely, as they are already in the plugins archived repo (read mode).